internal/blockchain/v0/reactor.go

```diff
@@ -97,6 +100,8 @@ type Reactor struct {
 	// requestRoutine spawned goroutines when stopping the reactor and before
 	// stopping the p2p Channel(s).
 	poolWG sync.WaitGroup
+
+	metrics *con.Metrics
```

Comment:
now this is a code smell lol. Why does the blockchain reactor have a reference to the consensus reactor's metrics?

Reply:
ah. Thanks, I will remove it.